packages/manager/modules/pci/src/constants.js

```diff
@@ -194,7 +194,8 @@ export const PCI_REDIRECT_URLS = {
     contacts: 'https://www.ovh.com/manager/dedicated/index.html#/useraccount/contacts?tab=SERVICES&serviceName={serviceName}',
     horizon: 'https://horizon.cloud.ovh.net/openstackdashboard?username={username}',
     ipAction: 'https://www.ovh.com/manager/dedicated/index.html#/configuration/ip?action={action}&ip={ip}&ipBlock={ipBlock}',
-    paymentMeans: 'https://www.ovh.com/manager/dedicated/index.html#/billing/mean',
+    paymentMeans: 'https://www.ovh.com/manager/dedicated/index.html#/billing/mean', // should not be used anymore
```

Comment:
If used only in `creating` it should be in a constant file inside `creating`.

Reply:
For the moment it is only used into pci.projects.new but I think this manager section will be used elsewhere in the future